packages/e2e-tests/entrypoint.sh

```diff
@@ -1,12 +1,23 @@
-#!/bin/sh
-
-# Export some variables
-export DISPLAY=:99.0
-export PUPPETEER_EXEC_PATH="google-chrome-stable"
+#!/bin/bash
 
 # Startup Xvfb
 Xvfb -ac :99 -screen 0 1280x800x24 -ac -nolisten tcp -dpi 96 +extension RANDR > /dev/null 2>&1 &
 
-cd packages/e2e-tests
-set -e
-exec "$@"
\ No newline at end of file
+if [[ $@ == *"bash"* ]]
+then
```

Comment:
@spersico Nice work! FYI, you can also override entrypoint via a command line flag. So this is not strictly necessary.